geowave-test/src/main/java/mil/nga/giat/geowave/test/service/ServicesTestEnvironment.java

```diff
@@ -96,8 +103,28 @@ public static void startServices() {
 					gsWebapp.setContextPath(GEOSERVER_CONTEXT_PATH);
 					gsWebapp.setWar(GEOSERVER_WAR_DIR);
 
-					final WebAppClassLoader classLoader = new WebAppClassLoader(
-							gsWebapp);
+					final WebAppClassLoader classLoader = AccessController.doPrivileged(new PrivilegedAction<WebAppClassLoader>() {
```

Comment:
Apparently all classloader instances should be instantiated inside a doPrivileged block.